browser_history/cli.py

```diff
@@ -44,7 +44,7 @@ def main():
     else:
         try:
             # gets browser class by name (string). TODO: make it case-insensitive
-            browser_class = getattr(browsers, args.browser)
+            browser_class = getattr(browsers, args.browser.capitalize())
```

Comment:
Looks good, but this will fail when the browser name is made up of multiple words. For example, `ChromiumEdge` will fail is the argument passed is `chromiumedge`.